builtin/logical/cassandra/path_roles.go

```diff
@@ -54,6 +54,11 @@ template values are '{{username}}' and
 				Default:     "4h",
 				Description: "The lease length; defaults to 4 hours",
 			},
+			
+			"consistency": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "The consistency level for the operations.",
```

Comment:
There are two schools of thought here. The first is, document the default from gocql and trust gocql to use an appropriate default, with the caveat that if gocql changes it, the documentation here could be out of date.

The second is explicitly declare a `Default:` value and document that; then if gocql changes we can update this later but at least the behavior will match the description.

Personally I prefer the latter, but I see arguments for both.